common/seo.js

```diff
@@ -1,72 +1,60 @@
-import { SOURCES } from '@sttm/banidb';
-
-const suffix = suffix => (value = '') =>
-  `${value}${value === '' ? '' : ' - '}${suffix}`;
-
-const suffixAppName = suffix('SikhiToTheMax');
-
-const getAng = req => req.query.ang;
-const getSource = req => SOURCES[req.query.source || 'G'];
-const basicDescription = `Search Sri Guru Granth Sahib Jee, Sri Dasam Granth Sahib, Bhai Gurdas Vaar, Bhai Nandlal Vaar, Bhai Gurdas Singh Vaar, Tankhanaama, Zafarnaama, Amrit Kirtan.`;
+import { suffixAppName } from '../server/utils/';
+import { BASIC_META_DESCRIPTION } from './constants';
 
 export default {
   '/': {
-    title: suffixAppName(),
+    createTitle: () => suffixAppName(),
     createDescription: () =>
-      `SikhiToTheMax, online Gurbani Searcher. ${basicDescription}`,
+      `SikhiToTheMax, online Gurbani Searcher. ${BASIC_META_DESCRIPTION}`,
   },
   '/404': {
-    title: suffixAppName('Page not found'),
-    createDescription: () => `Page not found. ${basicDescription}`,
+    createTitle: () => suffixAppName('Page not found'),
+    createDescription: () => `Page not found. ${BASIC_META_DESCRIPTION}`,
   },
   '/terms-of-service': {
-    title: suffixAppName('Terms Of Service'),
+    createTitle: () => suffixAppName('Terms Of Service'),
     createDescription: () =>
-      `Terms of Service of SikhiToTheMax. ${basicDescription}`,
+      `Terms of Service of SikhiToTheMax. ${BASIC_META_DESCRIPTION}`,
```

Comment:
Don't add BASIC_META_DESCRIPTION here. It might have negative SEO impact (there's a length limit for description) and will create unnecessary noise.